src/beyondllm/generator/generate.py

```diff
@@ -72,6 +66,11 @@ def pipeline(self):
             If you FAIL to execute this task, you will be fired and you will suffer 
             """
 
+    def call(self, question: str) -> str:
```

Comment:
we shouldn't directly remove this. We need to give a deprecation warning. If someone uses question inside generate, then we need show a warning message to tell them use call()